Kitodo-Query-URL-Import/src/main/java/org/kitodo/queryurlimport/QueryURLImport.java

```diff
@@ -279,7 +279,12 @@ private DataRecord performQueryToRecord(String queryURL, String identifier) thro
             }
         }
         String prefix = Objects.nonNull(idPrefix) && !identifier.startsWith(idPrefix) ? idPrefix : "";
-        fullUrl += encodeQueryParameter(idParameter + equalsOperand + prefix + identifier, encoding);
+        String queryParameter = idParameter + equalsOperand + prefix + identifier;
+        if (Objects.nonNull(interfaceType) && SearchInterfaceType.SRU.equals(interfaceType)) {
```

Comment:
Unnecessary `nonNull()` check. Enum constants compare to `null` as `false` and don’t throw `NullPointerException`.